### PR TITLE
Edit syncs with edit sync wizard

### PR DESCRIPTION
**Pull request by yeldah3**
https://www.loom.com/share/d32d0dd5582e4908b6624ab7ba65bf11

Screenshot 2024-05-24 at 2 35 09 PM

### CODE REVIEW

**Comment by yeldah3** [comment]
* **#39** <a href="https://app.graphite.dev/github/pr/sutrolabs/census-embedded-demo/39?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @yeldah3 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: components/SyncObject.js
Comment (yeldah3) [inline components/SyncObject.js:1]: Recommend reviewing this file without whitespace!
Screenshot 2024-05-24 at 2 33 47 PM

Comment (LemonFace0309) [reply components/SyncObject.js:1]: ty!!!

File: components/SyncObject.js [hunk 1]
```diff
@@ -103,109 +134,148 @@ export function SyncObject({
...
-              {
-                ["paused"]: !sync.paused,
-              },
-              null,
-              2,
-            )}
-          </pre>
-        }
-      />
+      {!editSyncWizardLink && (
```
Comment (LemonFace0309) [inline components/SyncObject.js:195]: were these randomly popping up when moving the mouse around if were in edit sync wizard mode? Why only show them if `!editSyncWizardLink`?

**Review by LemonFace0309 — APPROVED** [review]
LET'S GOOOO

File: components/SyncObject.js (hunk 1 above)
Comment (yeldah3) [reply components/SyncObject.js:195]: Great point! I'm not sure if this check is needed. Mainly we hide the delete/edit/pause/run buttons when showing the sync wizard, so I wasn't sure if the tooltips would be mad about their anchors being hidden. I'll check to see if anything's actually broken without this check

Comment (yeldah3) [reply components/SyncObject.js:195]: You right, this check does not seem to be needed 👍 yeeting!

Comment (LemonFace0309) [reply components/SyncObject.js]: woohoo!

**Pull request merged**